instrumentation/play-shaded-async-http-client-1.0.0/build.gradle

```diff
@@ -18,28 +18,17 @@ verifyInstrumentation {
 }
 
 test {
```

Comment:
I'm guessing that you'd prefer these be fixed in a separate branch.

Reply:
Oh, I'll address those in this PR to see if the commented out code works as expected. If so, it would greatly clean things up.